### ISSUE TITLE
Parrot-CVMFS Multiple Repo Warning?

### ISSUE THREAD

**Issue by dthain** [cooperative-computing-lab/cctools#1822]
The current version of parrot still spits out an error about switching repositories,
but I believe this problem was solved some time ago, yes?

```
 dthain@cclws15~$ setenv HTTP_PROXY ";DIRECT"
[dthain@cclws15 ~]$ cd /cvmfs/oasis.opensciencegrid.org
[dthain@cclws15 oasis.opensciencegrid.org]$ ls
atlas  cmssoft  enmr      geant4  gluex  ligo  nanohub  osg           sbgrid
auger  csiu     fermilab  glow    ilc    mis   nova     osg-software  snoplussnolabca
[dthain@cclws15 oasis.opensciencegrid.org]$ cd /cvmfs/cms.cern.ch
2018/01/15 11:01:47.95 parrot_run[23142] <child:23143> notice: ERROR: using multiple CVMFS repositories in a single parrot session is not allowed.  Define PARROT_ALLOW_SWITCHING_CVMFS_REPOSITORIES to enable experimental support, which could result in parrot crashing or performing poorly.
```

**Comment by btovar** [comment]
Yes.

**Comment by btovar** [comment]
Fixed by #1823
